### PR TITLE
Data folder loc

### PR DESCRIPTION
**Pull request by hayfield**
This allows all data about a single version of the Standard to be accessed at once.

Content around Codelist mapping files has been removed. This may be re-added at some point in the future.

### CODE REVIEW

**Comment by hayfield** [comment]
The second commit is not hugely related to this branch, though is here to trigger a Travis build (Travis wasn't set up on the first commit and there was no obvious other way to trigger a build).

File: iati/core/resources.py [hunk 1]
```diff
@@ -26,22 +26,25 @@
...
-BASE_PATH_CODELISTS_EMBEDDED = os.sep.join((BASE_PATH_CODELISTS, 'embedded'))
-"""The relative location of the folder containing embedded codelists from the SSOT."""
-BASE_PATH_CODELISTS_NON_EMBEDDED = os.sep.join((BASE_PATH_CODELISTS, 'non_embedded'))
-"""The relative location of the folder containing non-embedded codelists from the SSOT."""
-BASE_PATH_DATA = os.sep.join((BASE_PATH, 'data'))
+BASE_PATH_202 = os.sep.join((BASE_PATH, '202'))
+"""The relative location of the resources folder for version 2.02 of the IATI Standard."""
+PATH_CODELISTS = 'codelists'
+"""The location of the folder containing codelists from the SSOT."""
+PATH_CODELISTS_EMBEDDED = os.sep.join((PATH_CODELISTS, 'embedded'))
```
Comment (dalepotter) [inline iati/core/resources.py]: Could potentially change the defaults to be the repo name on the IATI Github - i.e `IATI-Codelists` in this case

File: iati/core/resources.py [hunk 2]
```diff
@@ -26,22 +26,25 @@
...
+"""The location of the folder containing embedded codelists from the SSOT."""
+PATH_CODELISTS_NON_EMBEDDED = os.sep.join((PATH_CODELISTS, 'non_embedded'))
```
Comment (dalepotter) [inline iati/core/resources.py]: As with the previous comment, this could be `IATI-Codelists-NonEmbedded`

File: iati/core/resources.py (hunk 1 above)
Comment (hayfield) [inline iati/core/resources.py]: ie. changing `embedded` to `IATI-Codelists`?

If so, there are a few problems (all small, but probably worth noting) with straight-up using that:

* It's mixed case, which could cause some problems depending on the underlying file system.
* The names and organisation of the git repos are a bit not nice, and use `IATI` more than they probably should.
* It's inconsistent with the folder names for Schemas / Rulesets (unless they're changed too).

File: iati/core/resources.py (hunk 2 above)
Comment (hayfield) [inline iati/core/resources.py]: See above.

Also, should hyphens or underscores be used as a separator in folder / file names?

File: iati/core/resources.py (hunk 1 above)
Comment (dalepotter) [inline iati/core/resources.py]: All fair points - was just thinking re consistency with the existing names.

**Pull request merged**